x-pack/plugins/gis/public/shared/layers/styles/vector_style.js

```diff
@@ -45,7 +45,7 @@ export class VectorStyle {
     return 'Vector style';
   }
 
-  static description = 'Link styles to property values for data driven styling.';
+  static description = 'Use property values to symbolize features.';
```

Comment:
I thought there was more to this like diplaying the `link` button?

Reply:
`symbolize features` feels so vague to me. I have no idea what to expect to happen.

Reply:
discussed this with @gchaps 

It's kind of "wrong" too, since the symbolization is not just dynamic. My preference would be to just remove it and have a tooltip on the link-paperclip icon.